bdd-generator/pom.xml

```diff
@@ -178,6 +178,12 @@
             <version>3.4.1</version>
         </dependency>
 
+        <dependency>
```

Comment:
Can we avoid this dependency?
Spring already contains org.springframework.boot.liquibase.CommonsLoggingLiquibaseLogger
which redirects liquibase logs to org.apache.commons.logging.Log (package spring-jcl)
and org.apache.commons.logging.impl.SLF4JLog (package jcl-over-slf4j)

I believe if we exclude commons-logging from jira-client dependency, we might achieve same result without introducing new dependency. You can consult me or Maksim on exclusion topic.

Reply:
@hdghg Done. I removed the dependency.